internal/ls/completions.go

```diff
@@ -1907,128 +1920,29 @@ func (l *LanguageService) createCompletionItem(
 		// Otherwise use the completion list default.
 	}
 
-	// Text edit
```

Comment:
This all got moved to `createLSPCompletionItem` so it can be reused in other places that also created completion entries and now need to create an LSP completion item.